### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Nathan Wulliamoz - Benjamin Kocher

### DISCUSSION THREAD

**Discussion by BKo1706** [heig-vd-dai-course/heig-vd-dai-course#553] [Show and tell]
Notre API permettra de gérer une liste de films et de séries avec des informations complètes (synopsis, réalisateur, acteurs, affiche) avec afin que ce soit une CRUD API, la possibilité d'ajouter, de consulter, de modifier et de supprimer des films et series.

**Comment by ludelafo** [comment]
OK pour nous !

**Comment by nathan-heig** [comment]
Bonjour,
Voici le lien du répo Github: https://github.com/nathan-heig/DAI_lab03.git
Voici le hash du dernier commit: e37a0fa4ba4c7be010932078e75a5e71dc476250
Bonne journée

**Reply by ludelafo** [reply]
Bonjour @nathan-heig,

Je n'ai pas réussi à accéder à <agooddailab.duckdns.org>. Est-ce que votre serveur tourne ?

Merci d'avance,
Ludovic

**Comment by nathan-heig** [comment]
Bonjour, oui le serveur tourne toujours vous voulez accéder en ssh ? Pour l'api il faut utiliser le sous domaine webapp.agooddailab.duckdns.org et pour accéder au dashboard traefik.agooddailab.duckdns.org. Votre à été ajouté hier pour le ssh et devrait fonctionner si c'est la bonne (ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIF5deyMbQaRaiO4ojymkCoWBtwPyG8Y+4BbLQsb413KC heig-vd-dai-course)
________________________________
De : Ludovic Delafontaine ***@***.***>
Envoyé : lundi, 20 janvier 2025 13:40
À : heig-vd-dai-course/heig-vd-dai-course ***@***.***>
Cc : Wulliamoz Nathan ***@***.***>; Mention ***@***.***>
Objet : Re: [heig-vd-dai-course/heig-vd-dai-course] DAI 2024-2025 - Practical work 3 - Nathan Wulliamoz - Benjamin Kocher (Discussion #553)


Bonjour @nathan-heig<https://github.com/nathan-heig>,

Je n'ai pas réussi à accéder à <agooddailab.duckdns.org>. Est-ce que votre serveur tourne ?

Merci d'avance,
Ludovic

**Reply by ludelafo** [reply]
Bonjour,

Merci pour votre retour, je confirme avoir bien accès en utilisant les sous-domaines. J'utilisais le domaine principal, c'est sans doute pour cela que ça ne marchait pas.

Merci encore !

**Comment by nathan-heig** [comment]
J'ai oublié de préciser mais pour accéder au dashboard un user et un password sont demandé. L'identifiant est user:password et les dashboard uniquement accessible via https
________________________________
De : Ludovic Delafontaine ***@***.***>
Envoyé : lundi, 20 janvier 2025 13:54
À : heig-vd-dai-course/heig-vd-dai-course ***@***.***>
Cc : Wulliamoz Nathan ***@***.***>; Mention ***@***.***>
Objet : Re: [heig-vd-dai-course/heig-vd-dai-course] DAI 2024-2025 - Practical work 3 - Nathan Wulliamoz - Benjamin Kocher (Discussion #553)


Bonjour,

Merci pour votre retour, je confirme avoir bien accès en utilisant les sous-domaines. J'utilisais le domaine principal, c'est sans doute pour cela que ça ne marchait pas.

Merci encore !

**Reply by ludelafo** [reply]
Merci bien !
